packages/expo-updates/src/UpdatesProviderUtils.ts

```diff
@@ -0,0 +1,93 @@
+import * as Updates from './Updates';
+import type { Manifest, UpdateEvent } from './Updates.types';
+import type {
+  AvailableUpdateInfo,
+  UpdatesInfo,
+  UpdatesProviderCallbacksType,
+} from './UpdatesProvider.types';
+import { currentlyRunning } from './UpdatesProviderConstants';
+
+/////// Internal functions ////////
+
+// Constructs the availableUpdate from the update manifest
+export const availableUpdateFromManifest = (manifest: Manifest | undefined) => {
+  return manifest
+    ? {
+        updateId: manifest?.id ? manifest?.id : null,
+        createdAt: manifest?.createdAt ? new Date(manifest?.createdAt) : null,
```

Comment:
In classic manifests, this is `publishedTime`

Reply:
Ok I finally found the reference to `publishedTime` in `expo-constants`. For some reason, the type is hidden from `expo-updates`, so I'll have to figure out how to expose that, or copy that into this source. Also I'm not sure we should be supporting classic updates with this feature :)

Reply:
Fixed